ci/cudf_pandas_scripts/pandas-tests/run.sh

```diff
@@ -27,9 +27,10 @@ bash python/cudf/cudf/pandas/scripts/run-pandas-tests.sh \
   --dist worksteal \
   --report-log=${PANDAS_TESTS_BRANCH}.json 2>&1
 
+SUMMARY_FILE_NAME=${PANDAS_TESTS_BRANCH}-24.06-results.json
```

Comment:
If this version has to be hard coded here, then an entry needs to be added to `ci/release/update-version.sh`.

Reply:
Yup, just updated.